test/run_tests.sh

```diff
@@ -20,4 +20,4 @@ pytest \
     --cov-report=html \
     --cov-report=xml \
     -vv \
-    test/
+    test/AssemblySet_basic_test.py test/DifferentialExpressionMatrixSet_basic_test.py
```

Comment:
Just adding in the ones that are fixed up for now? Watching that test coverage grow after each PR will be nice. :)